### ISSUE TITLE
Regression in 7.2.0 - basePath missing from URL in UI

### ISSUE THREAD

**Issue by peterbroadhurst** [mrin9/RapiDoc#123]
With `<script src="https://unpkg.com/rapidoc@7.1.1/dist/rapidoc-min.js"></script>` we see the following correct URL in the browser, and the exerciser works correctly:

![image](https://user-images.githubusercontent.com/6660217/73620211-cf4a8a80-45fe-11ea-8060-4e2b4170f85c.png)

Whereas with latest (or `<script src="https://unpkg.com/rapidoc@7.2.0/dist/rapidoc-min.js"></script>`) we see the URL loses the `basePath` and the exerciser is broken:

![image](https://user-images.githubusercontent.com/6660217/73620183-a88c5400-45fe-11ea-8b9c-ec4d52de6750.png)

I'm working to isolate the code change, but as this is a potentially regression for other users I wanted to get it raised while I investigate.

**Comment by peterbroadhurst** [comment]
The regression is caused by this change https://github.com/mrin9/RapiDoc/commit/504af8a5eaaa8f1b995e0522bf8ac155feb1dcf4

Re-instating `json-refs` in `spec-parser.js` instead of `swagger-client` fixes the problem.
Digging further to see what's different about the `jsonParsedSpec` output from the two tools.

**Comment by peterbroadhurst** [comment]
I believe the bug is a one-liner:
https://github.com/mrin9/RapiDoc/blob/8ecfb4d778064b441660e7af51f78078eee5bc66/src/utils/spec-parser.js#L22-L26

See the bottom line overrides the output from `convertedSpec.openapi` back to `specObj.spec` in the case of a Swagger 2.0 -> OpenAPI 3.0 conversion.

If I simply put the `jsonParsedSpec = specObj.spec` before the if block, the problem is resolved.

```js
    jsonParsedSpec = specObj.spec;
    if (specObj.spec.swagger) {
      convertedSpec = await converter.convertObj(specObj.spec, convertOptions);
      jsonParsedSpec = convertedSpec.openapi;
    }
```

**Comment by peterbroadhurst** [comment]
I've submitted a PR - fyi @mrin9 in case you're able to review and generate a release for this.
I think this will hit anyone using the hosted latest RapiDoc since 7.2.0 was pushed, who have Swagger 2.0 definitions.

**Comment by mrin9** [comment]
thanks a lot for spotting it early and the quick PR.
did a bug-fix release `7.2.1` and added a test case to avoid this in future
